spec/models/miq_schedule_spec.rb

```diff
@@ -512,7 +512,7 @@
     end
 
     context "valid action_automation_request" do
-      let(:admin) { FactoryBot.create(:user_miq_request_approver) }
+      let(:admin) { FactoryBot.create(:user_miq_request_approver, :userid => 'admin') }
```

Comment:
pretty sure you don't want `:userid` here
you may need a before with `User.seed`